components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -197,6 +202,11 @@ func main() {
 			step:     provisioning.NewProvideLmsTenantStep(lmsTenantManager, db.Operations()),
 			disabled: cfg.LMS.Disabled,
 		},
+		{
+			weight:   1,
+			step:     provisioning.NewEDPRegistration(db.Operations(), edpClient, cfg.EDP),
```

Comment:
I'd suggest to rename it to be consistent with convention
```suggestion
			step:     provisioning.NewEDPRegistrationStep(db.Operations(), edpClient, cfg.EDP),
```